client/reader/start/style.scss

```diff
@@ -81,14 +81,26 @@
 		font-family: $serif;
 		font-size: 12px;
 		margin: 10px 0 -3px;
-		max-height: none;
-	}
+		max-height: 65px;
 
-	.post-excerpt.is-long {
+		@include breakpoint( ">480px" ) {
+			max-height: 56px;
+		}
 
 		&:after {
-			display: none;
+			@include long-content-fade;
+			height: 25px;
+			position: absolute;
+				bottom: 0;
+				top: 40px;
+			width: 65%;
```

Comment:
I was trying to conceal the ellipsis that appears on some cards in Chrome, but I guess it isn't too bad: https://github.com/Automattic/wp-calypso/pull/6186/commits/ac0d5417619cae08b87ca56f4f6b26319dd2f751